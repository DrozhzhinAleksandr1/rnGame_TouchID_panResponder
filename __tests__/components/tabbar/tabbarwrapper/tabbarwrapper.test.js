import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {TabBarWrapper} from '../../../../app/components/tabbar/tabbarwrapper/tabbarwrapper';

it('create snapshot WarningMessage', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<TabBarWrapper />);
    expect(toolbarSnapshot).toMatchSnapshot();
});