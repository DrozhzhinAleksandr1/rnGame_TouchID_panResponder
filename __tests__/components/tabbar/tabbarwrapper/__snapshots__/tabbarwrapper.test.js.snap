// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot WarningMessage 1`] = `<withOrientation />`;
